service/src/test/java/org/jboss/aerogear/unifiedpush/service/ClientInstallationServiceTest.java

```diff
@@ -101,6 +96,100 @@ public void registerDevices() {
     }
 
     @Test
+    public void registerDevicesWithCategories() {
+
+        Installation device = new Installation();
+        String deviceToken = generateFakedDeviceTokenString().toUpperCase();
+        device.setDeviceToken(deviceToken);
+        final Set<Category> categories = new HashSet<Category>(Arrays.asList(new Category("football"), new Category("football")));
```

Comment:
This will result in one category, because it's a Set, do we really want to write it like that?